pkg/utils/testing.go

```
@@ -297,8 +297,9 @@ func (p *pfMockNetlinkLib) LinkDelAltName(link netlink.Link, name string) error
 
 func (p *pfMockNetlinkLib) recordMethodCall(format string, a ...any) {
```

golangci-lint (2.5.0)

[error] printf-like formatting function 'recordMethodCall' should be named 'recordMethodCallf'

(goprintffuncname)